feat(*): New `Get QuestDB` page

insmac opened the pull request.

goodroot commented:
Nice one! This page sure fills a gap...

Did not alter matrix until we're aligned on format in Slab.

A couple minor visual things:

---

Contrast on green version text on light mode is bright.

<img width="1036" alt="Screenshot 2023-08-29 at 10 15 35 AM" src="https://github.com/questdb/questdb.io/assets/9484709/18c1e4a0-2d9a-464f-9058-ec0acd3cdc27">

Also, on light, the 3-pack feels very "in the open".

Would we consider a container, like with these?

<img width="1306" alt="Screenshot 2023-08-29 at 10 21 16 AM" src="https://github.com/questdb/questdb.io/assets/9484709/4d4bdc89-9d69-488f-ba77-7d44ba8504b3">

---
I'm trying to find a way to point out "no CC trial". Its not something I see going in the descriptions, and I don't want to cram the _$200 free credit_ pill.

Any thoughts?

<img width="334" alt="Screenshot 2023-08-29 at 10 32 55 AM" src="https://github.com/questdb/questdb.io/assets/9484709/04f963f8-3c60-4512-adbc-cff1d3369c82">

---

There are two reasons that I think the `/open-source/` page may be better as `/downloads`: 

* can see "open source" getting its own landing page at some point
* SEO -- "download questdb" is a popular query that we expect will increase -- hitting that "on the nose" with a download page would serve us well.

---

Also much appreciate the quick work getting this page in code. 🔥

insmac commented:
@goodroot Addressed the issues with the light theme, and swapped the URL to `/download/`. 

As for "no card required", I propose the following approach:
<img width="1289" alt="CleanShot 2023-08-30 at 12 23 08@2x" src="https://github.com/questdb/questdb.io/assets/1871646/7f749970-4eb4-41ed-a0cb-d1abbf884d11">

To maintain symmetry, I have moved the blue links below the description, but above the main CTA, so that all content rows are equally distributed.

Let me know what you think 🙂 (the above is a screenshot from Figma)

goodroot commented:
@insmac Nice one!

I'd say it's worth putting the product version in navbar + the altered button in code.
